### PR TITLE
TPC: Multiple fixes and extensions

### PR DESCRIPTION
**Pull request by wiechula**

### CODE REVIEW

**Comment by wiechula** [comment]
It seems the error is not related to this PR, could someone please have a look?

File: Detectors/TPC/workflow/src/CalibratordEdxSpec.cxx [hunk 1]
```diff
@@ -155,9 +205,11 @@ DataProcessorSpec getCalibratordEdxSpec(const o2::base::Propagator::MatCorrType
     outputs,
     adaptFromTask<CalibratordEdxDevice>(ccdbRequest, matType),
     Options{
-      {"tf-per-slot", VariantType::UInt32, 6000u, {"number of TFs per calibration time slot"}},
+      {"tf-per-slot", VariantType::UInt32, 6000u, {"number of TFs per calibration time slot, is overwritten by time-slot-seconds if > 0"}},
+      {"time-slot-seconds", VariantType::Int, 300, {"seconds per calibration time slot, overwrites tf-per-slot if > 0"}},
```
Comment (shahor02) [inline Detectors/TPC/workflow/src/CalibratordEdxSpec.cxx]: This device level option is the same as the workflow level option of the `Detectors/TPC/workflow/src/tpc-vdrift-tgl-calibration-workflow.cxx`, this is not supported by the DPL:
```
[67118:tpc-vd-tgl-calib]: [ERROR] error parsing options of o2-tpc-vdrift-tgl-calibration-workflow: option '--time-slot-seconds' is ambiguous and matches different versions of '--time-slot-seconds'
```
One should either have both on the same level or different names.

**Review by shahor02 — CHANGES_REQUESTED** [review]
Hi @wiechula 
the fullCI failure is genuine, see below.

File: Detectors/TPC/workflow/src/CalibratordEdxSpec.cxx (hunk 1 above)
Comment (wiechula) [reply Detectors/TPC/workflow/src/CalibratordEdxSpec.cxx]: Thanks @shahor02 , indeed I overlooked this.

**Comment by wiechula** [comment]
Thanks for checking @shahor02 , can this be merged now?

**Pull request merged**